src/components/TaskList.vue

```diff
@@ -15,12 +15,10 @@
       <Task
         v-for="task in tasks"
         :key="task.taskId"
-        :taskId="task.taskId"
         :taskState="task.taskState"
         :taskText="task.taskText"
-        :listId="listId"
-        @removeTask="onRemoveTask(task.taskId)"
-        @changeTaskState="onChangeTaskState(task.taskId)"
+        @removeTask="onRemoveTask(task.taskId, listId)"
+        @changeTaskState="onChangeTaskState(task.taskId, listId)"
```

Comment:
It not necessary send **listId** property from template, because it is available inside component method like **this.listid**